Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -170,7 +170,7 @@ script:
     - auto: PREDEFINED
       defaultValue: incident
       description: Ticket type. Can be "incident", "problem", "change_request", "sc_request",
-        "sc_task", or "sc_req_item". Default is "incident".
+        "sc_task", "sc_req_item" or "sn_si_incident". Default is "incident".
```

Comment:
please fix in all other places where it was removed (search for `Ticket type. Can be "incident"`, there are 6) as well